src/main/java/com/laserfiche/repository/api/clients/impl/model/APIServerException.java

```diff
@@ -2,26 +2,175 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.util.Objects;
 
 @javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaClientCodegen", date = "2022-08-19T17:07:43.799-04:00[America/New_York]")
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class APIServerException {
 
     @JsonProperty("operationId")
-    public String operationId = null;
+    private String operationId = null;
 
     @JsonProperty("message")
-    public String message = null;
+    private String message = null;
 
     @JsonProperty("errorCode")
-    public Integer errorCode = null;
+    private Integer errorCode = null;
 
     @JsonProperty("errorClass")
-    public String errorClass = null;
+    private String errorClass = null;
 
     @JsonProperty("statusCode")
-    public Integer statusCode = null;
+    private Integer statusCode = null;
 
     @JsonProperty("errorSource")
-    public String errorSource = null;
+    private String errorSource = null;
+
+    public APIServerException operationId(String operationId) {
+        this.operationId = operationId;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getOperationId() {
+        return operationId;
+    }
+
+    public void setOperationId(String operationId) {
+        this.operationId = operationId;
+    }
+
+    public APIServerException message(String message) {
```

Comment:
It is not clear for me why do we need methods like **message** in addition to the corresponding setters/getters.

Reply:
Yes. I think it is needed, as having this kind of method is not common in java.
Maybe we can create a new user story for removing such methods.